pkg/graph/schema.graphql

```diff
@@ -762,7 +762,7 @@ type Query {
     after: String
     first: Int!
   ): AccessibilityRequestsConnection
-  grtFeedbacks(intakeID: UUID!): [GRTFeedback!] @hasRole(role: EASI_GOVTEAM)
+  grtFeedbacks(intakeID: UUID!): [GRTFeedback!]
```

Comment:
I wrote the same question this morning but evidently failed to hit submit in that browser window. I do think putting this on `systemIntake` would be better since it would let us grab these along with the system intake if we needed.